apps/cowswap-frontend/src/modules/swap/pure/Row/RowSlippageContent/index.tsx

```diff
@@ -105,38 +86,21 @@ export function RowSlippageContent(props: RowSlippageContentProps) {
     <StyledRowBetween {...styleProps}>
       <RowFixed>
         <TextWrapper>
-          {showSettingOnClick ? (
-            <ClickableText onClick={toggleSettings}>
-              <SlippageTextContents
-                isEoaEthFlow={isEoaEthFlow}
-                slippageLabel={slippageLabel}
-                isDynamicSlippageSet={isSmartSlippageApplied}
-              />
-            </ClickableText>
-          ) : (
-            <SlippageTextContents
-              isEoaEthFlow={isEoaEthFlow}
-              slippageLabel={slippageLabel}
-              isDynamicSlippageSet={isSmartSlippageApplied}
-            />
-          )}
+          <SlippageTextContents
+            isEoaEthFlow={isEoaEthFlow}
+            slippageLabel={slippageLabel}
+            isDynamicSlippageSet={isSmartSlippageApplied}
+          />
```

Comment:
I was thinking about just killing it because of https://stackoverflow.com/questions/70596487/how-to-programmatically-expand-react-reach-ui-reach-menu-button-menu

But eventually added this functionality back: https://github.com/cowprotocol/cowswap/pull/4950/commits/16785573a04dccc05c068634c8a04de451b08c2a